frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -293,7 +293,9 @@ pods:
           {{#SECURITY_KERBEROS_ENABLED}}
           export HADOOP_OPTS="-Djava.security.krb5.conf=$MESOS_SANDBOX/{{TASKCFG_ALL_HDFS_VERSION}}/etc/hadoop/krb5.conf $HADOOP_OPTS" &&
           {{/SECURITY_KERBEROS_ENABLED}}
-          ./bootstrap --resolve-hosts $TASK_NAME.$FRAMEWORK_HOST && ./{{TASKCFG_ALL_HDFS_VERSION}}/bin/hdfs zkfc
+          ./bootstrap --resolve-hosts $TASK_NAME.$FRAMEWORK_HOST &&
```

Comment:
`--resolve-hosts` accepts a comma-separated list of hosts, so the name node can be added to the list instead of requiring an additional call to `bootstrap`.

Reply:
Ah yes, it seems so